src/common/hook/usePublishQueue/usePublishQueue.test.tsx

```diff
@@ -62,6 +62,6 @@ describe("usePublishQueue", () => {
     await act(async () => {
```

Comment:
Can you add  `mockPublishJob.mockResolvedValue("tx-id");` before line 60? 
Notice that this test is not independent, running only this test will result in test failure because it was waiting for the previous test to set it up

Reply:
ok